### PR TITLE
24 Update set up files

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
When I ran `pkgdown::build_site()` it doesn't generate the README.md file using README.Rmd. I also getting a NOTE when running `devtools::document()` about a time stamp not being found??
Any help appreciated!

**Comment by bundfussr** [comment]
I think `README.md` is generated by the "Render README" workflow. If you want to create it locally, you could try `rmarkdown::render("README.Rmd", output_format = "md_document")`. I have not tried it but it's what the workflow does.

**Comment by millerg23** [comment]
@bundfussr @cicdguy would you say we can accept the LINKs failing for now? Think everything else is working OK

**Comment by millerg23** [comment]
@bundfussr  you happy for me to merge this? Do you see anything I missed? Thanks!

File: .Rhistory [hunk 1]
```diff
@@ -1,35 +1,41 @@
+renv::status()
 renv::restore()
```
Comment (bundfussr) [inline .Rhistory:2]: This file should be removed.

File: _pkgdown.yml [hunk 2]
```diff
@@ -1,39 +1,71 @@
...
+search:
+  exclude: ['news/index.html']
 repo:
   url:
-    home: https://github.com/pharmaverse/admiraltemplate
-    source: https://github.com/pharmaverse/admiraltemplate/blob/main/
-    issue: https://github.com/pharmaverse/admiraltemplate/issues/
+    home: https://github.com/pharmaverse/admiralonco
+    source: https://github.com/pharmaverse/admiralonco/blob/test/
+    issue: https://github.com/pharmaverse/admiralonco/issues/
```
Comment (bundfussr) [inline _pkgdown.yml:13]: It should be `admiralophta` instead of `admiralonco` and `main` instead of `test` for `source:`.

**Review by bundfussr — CHANGES_REQUESTED** [review]
- `.Rbuildignore` should be updated. There are references to admiralext. And the entries for the vignettes do not make sense.
- `Authors@R:`, `Description:`, and `URL:` in `DESCRIPTION` should be updated.
- I would remove all the entries in `R/globals.R` because it will be hard to remove the unnecessary ones later.
- I would remove `R/my_first_fnc.R`, `man/hello_admiral.Rd`, and `tests/testthat/test-my_first_fcn.R`.

**Comment by millerg23** [comment]
@bundfussr thanks for the great feedback, I think I have made all the updates. Also added the information to generate README in a feature branch (copied from admiraltemplate).

**Review by bundfussr — APPROVED** [review]

**Pull request merged**